node-feature-discovery-0.15.yaml

```diff
@@ -0,0 +1,94 @@
+package:
+  name: node-feature-discovery-0.15
+  version: 0.15.4
+  epoch: 0
+  description: Node feature discovery for Kubernetes
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - protoc
```

Comment:
This was only used at buildtime. This should be able to be removed

Reply:
Removed!